bin/release-bom

```diff
@@ -28,32 +28,45 @@ fi
 
 version="$1"
 repo_url="https://updates.astronomer.io/astronomer-software/releases"
+bucket_path="gs://updates.astronomer.io/astronomer-software/releases"
 json_file_name="astronomer-${version}.json"
-local_json_file_path="./astronomer_bom_generator/doc/${json_file_name}"
+local_json_file_path="${json_file_name}"
 max_age="300"
 
 # Step 1: Runs the Python extractor script to generate the BOM JSON file
 echo "Running the extractor script to generate ${json_file_name}..."
 pip install "git+https://${GITHUB_TOKEN}@github.com/astronomer/astronomer-bom-generator.git@${BOM_GENERATOR_VERSION}"
 
+echo "Running the BOM generator script"
+bom-generator --version "${version}"
+
 if [ ! -f "${local_json_file_path}" ]; then
   echo "ERROR: Failed to generate ${json_file_name}"
   exit 1
 fi
 
+echo "Installing gcloud"
+sudo apt-get update
+sudo apt-get install apt-transport-https ca-certificates gnupg curl
+curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo gpg --dearmor -o /usr/share/keyrings/cloud.google.gpg
+curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key --keyring /usr/share/keyrings/cloud.google.gpg add -
+curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add -
+echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] https://packages.cloud.google.com/apt cloud-sdk main" | sudo tee -a /etc/apt/sources.list.d/google-cloud-sdk.list
+sudo apt-get update && sudo apt-get install google-cloud-cli
```

Comment:
We should not do this in this script. Instead, we should ensure that this CI step runs within a container that already has the necessary scripts set up, like quay.io/astronomer/ci-helm-release <https://github.com/astronomer/ci-images/blob/f8db5ecb69/helm-release/Dockerfile#L43>

Reply:
@danielhoherd I have added these libraries in circleci config, is that fine.?

Reply:
no, that's what i'm saying. we don't want to install these in the circleci pipeline. we want to run in an image that has these already installed.

Reply:
done